index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <link rel="stylesheet" href="index.css">
+    <title>skeleton mobile</title>
+</head>
+<body>
+    <header>
+      <!-- header navbar -->
+        <div class="mobile-container">
+            <!--top nav -->
+            <div class="topnav">
+                <!--hamburhuer tap-->
+                <a href="https://www.linkedin.com/in/roger-smith-a35738179/" class="active">John Doe</a>
+                <div id="myLinks">
+                    <a href="#Portafolio">Portafolio</a>
+                    <a href="#About">About</a>
+                    <a href="#Contact">contact</a>
+                </div>
+                <!--hamburguer tap-->
+
+                <nav>
+                    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                        <i class="fa fa-bars"></i>
+                    </a>
+                </nav>
+
+            </div>
+        </div>
+    </header>
+
+    <section>
+        <div class="roger">
+            
+                <div class="pad">
+                    <h1>Hey There.</h1>
+                    <h2 class="k">I´m Roger</h2>
+                </div>
```

Comment:
- [ ] Well done on implementing required changes from the other reviewer but kindly stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27). Keep shining 💪 

Here is a screenshot of your project:

![image](https://user-images.githubusercontent.com/74506933/158701419-a1d49546-c5f5-432d-ba10-d47c6a7dd52f.png)

Here is a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/158701734-ce3bb3d3-f06f-4a15-833d-64ff1de29f3a.png)

Reply:
@[Donard97](https://github.com/Donard97) hello coach you want me to change the name to alex